_includes/youtube_video.html

```diff
@@ -1 +1 @@
-<iframe class="youtube-iframe" align="center" width="{{ include.width }}" height="{{ include.height }}" src="{{ include.src }}" frameborder="0" allowfullscreen></iframe>
+<div class="youtube-container"><iframe class="youtube-iframe" align="center" src="{{ include.src }}?rel=0" frameborder="0" allowfullscreen></iframe></div>
```

Comment:
The URL may already have query parameters on it. The `rel=0` may be ignored by some URL parsers in some instance.